### PR TITLE
feat(3388): add presence

### PR DESCRIPTION
**Pull request by alexgokhale**
## Description 
Add presence for 3388 (https://3388.to).

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image
image
image
image
image
image

</details>

### CODE REVIEW

File: websites/0-9/3388/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"service": "3388",
+	"description": {
+		"en": "3388 is a streaming service for movies and TV."
+	},
+	"url": [
+		"3388.to",
+		"www.3388.to"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/FhGHKTn.png",
```
Comment (DarkVillager) [inline websites/0-9/3388/metadata.json:16]: Logo needs to be 512x512

Comment (DarkVillager) [inline websites/0-9/3388/metadata.json:16]: Logo needs to be 512x512
```suggestion
	"logo": "https://i.imgur.com/h8N7txd.png",
```

File: websites/0-9/3388/presence.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
+const presence = new Presence({
+	clientId: "1094931941616267344",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/FhGHKTn.png",
```
Comment (DarkVillager) [inline websites/0-9/3388/presence.ts:7]: Ditto.
```suggestion
			largeImageKey: "https://i.imgur.com/h8N7txd.png",
```

File: websites/0-9/3388/presence.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+			smallImageKey: "search",
```
Comment (DarkVillager) [inline websites/0-9/3388/presence.ts:8]: Also change these Assets to imgur links. If u use them alot use an enum.

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/0-9/3388/metadata.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "356827694572503051",
+		"name": "xela#0049"
```
Comment (theusaf) [inline websites/0-9/3388/metadata.json:5]:
```suggestion
		"name": "xela"
```

File: websites/0-9/3388/presence.ts [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+			coverElement.style.backgroundImage.endsWith('")')
+		) {
+			presenceData.largeImageKey = coverElement.style.backgroundImage.slice(
+				5,
+				coverElement.style.backgroundImage.length - 2
+			);
+		}
+
+		presenceData.details = `Viewing ${
+			!document.querySelector("#season-list") ? "a Movie..." : "a TV Show..."
```
Comment (theusaf) [inline websites/0-9/3388/presence.ts:36]: Remove "..." from strings. (Also remove from the lines at the bottom of this)
```suggestion
			!document.querySelector("#season-list") ? "a Movie" : "a TV Show"
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**